Kotlin/app/src/main/java/com/taboola/kotlin/examples/screens/web/WebComposeFeedFragment.kt

```diff
@@ -25,4 +28,27 @@ class WebComposeFeedFragment : Fragment() {
     }
 }
 
+/**
+ * This method adds WebView using Compose and integrate with Taboola SDK (Web Integration)
+ */
+@Composable
+fun CustomerWebView(placementInfo: PlacementInfo.WebFeedProperties) {
+
+    //This is the publisher code which displays a WebView on the screen
+    AndroidView(factory = { context ->
+        WebView(context).apply {
+            layoutParams = ViewGroup.LayoutParams(
+                ViewGroup.LayoutParams.MATCH_PARENT,
+                ViewGroup.LayoutParams.MATCH_PARENT
+            )
+
+            //Create taboolaWebWrapperViewModel instance
+            //Inside the WebView apply scope we will add Taboola code to use web integration
+            val taboolaWebWrapperViewModel = TaboolaWebWrapperViewModel()
```

Comment:
@voltfmanSlava - You don't create the view model inside of the AndroidView, but rather outside of it.

Reply:
The same goes for the other logic.

Reply:
Done

Reply:
@voltfmanSlava  - Not seeing that anything changed here

Reply:
Didn't pushed it yet.

Reply:
@voltfmanSlava  - You commented that it is done, so I assumed it was pushed.